src/serverlist.cpp

```diff
@@ -134,10 +132,7 @@ bool insert(const ServerListSpec &server)
 	std::string path = ServerList::getFilePath();
 	std::ostringstream ss(std::ios_base::binary);
 	ss << ServerList::serialize(serverlist);
-	if (!fs::safeWriteToFile(path, ss.str()))
-		return false;
-
-	return true;
+	return !fs::safeWriteToFile(path, ss.str()) ? true : false;
```

Comment:
This is the wrong way around. I suggest to revert this change. There's no point in touching this file just for rewriting the style of the condition.

Reply:
Will revert both these changes.